src/node_buffer.cc

```diff
@@ -1271,6 +1271,11 @@ void Initialize(Local<Object> target,
               Integer::NewFromUnsigned(env->isolate(), kMaxLength)).FromJust();
 
   target->Set(env->context(),
+              FIXED_ONE_BYTE_STRING(env->isolate(), "kMaxStringLength"),
```

Comment:
@bnoordhuis Huh, funny I didn’t see that. I used the other one anyway, so I’ve just removed the bits here again.